### PR TITLE
README updates and bug fix for Flask starter escaped quotes

### PR DESCRIPTION
**Pull request by bishopwm**
Updated README for Flask Python Starter with OAuth, fix bug with escaped quotes in Flask templates, add a yaml file to Connect Firebase example.

### CODE REVIEW

**Comment by horeaporutiu** [comment]
Hi @bishopwm - I've just tried this and all works fine. Great work! 

The one thing I would say is too call out that the dev needs to check the box for "expire auth token" when they create an app, otherwise the app won't work.

File: examples/python-flask-starter-with-oauth/README.md [hunk 1]
```diff
@@ -1,44 +1,80 @@
...
+# ✅ Prerequisites <a name="prerequisites"></a>
+
+- You have a [Miro account](https://miro.com/signup/).
+- You're [signed in to Miro](https://miro.com/login/).
+- Your Miro account has a [Developer team](https://developers.miro.com/docs/create-a-developer-team).
+- Your development environment includes [Node.js 14.13](https://nodejs.org/en/download) or a later version.
+- You have the latest versions of Flask and Python installed.
+
+# 🏃🏽‍♂️ Run the app locally <a name="run"></a>
+
```
Comment (horeaporutiu) [inline examples/python-flask-starter-with-oauth/README.md:38]: 0. Create a new Miro App and make sure to check the box for `expire authorization token` as shown in the screenshot below.

expire-auth-token

Comment (horeaporutiu) [inline examples/python-flask-starter-with-oauth/README.md:38]: 0. Create a new Miro App and make sure to check the box for `Expire user authorization token` as shown in the screenshot below. 

> ⚠️ If you don't check this box, the app will not work! ⚠️

 
expire-auth-token

**Review by horeaporutiu — APPROVED** [review]
Just left one suggestion - I think you should be able to directly apply it. 

Basically the app won't work unless you expire user auth token @bishopwm

File: examples/python-flask-starter-with-oauth/README.md (hunk 1 above)
Comment (bishopwm) [reply examples/python-flask-starter-with-oauth/README.md:38]: Great catch @horeaporutiu !

Just updated it with this 👍

**Review by janza — APPROVED** [review]

File: examples/python-flask-starter-with-oauth/README.md [hunk 2]
```diff
@@ -1,44 +1,86 @@
...
+0. Create a new Miro App and make sure to check the box for `Expire user authorization token` as shown in the screenshot below.
+
+   > ⚠️ If you don't check this box, the app will not work! ⚠️
```
Comment (mettin) [inline examples/python-flask-starter-with-oauth/README.md:41]: Can you explain why (in 1 sentence)?

**Review by mettin — APPROVED** [review]

**Pull request merged**